### PR TITLE
fix(image)!: remove lorempixel

### LINKED ISSUE #1571: Lorempixel is down
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [ ] I am willing to provide a PR.

### Describe the bug

The `faker.image` module generates URLs for a site [lorempixel.com](https://lorempixel.com). This site doesn't exist (presumably it was taken down)

### Minimal reproduction code

_No response_

### Additional Context

_No response_

### Environment Info

```shell
I was actually testing this on the json schema faker playground
```

### Which module system do you use?

- [ ] CJS
- [ ] ESM

### Used Package Manager

other

### PR DESCRIPTION
**Pull request by Shinigami92**
closes #1571

### CODE REVIEW

**Comment by Shinigami92** [comment]
/ping @matthewmayer please let me know (and approve) if the docs are enough

**Review by ST-DDT — DISMISSED** [review]

File: docs/guide/upgrading.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Not the version you are looking for?
 | `faker.address.citySuffix`      | _Removed_                                                         |
 | `faker.address.streetPrefix`    | _Removed_                                                         |
 | `faker.address.streetSuffix`    | _Removed_                                                         |
+| `faker.image.lorempixum`        | _Removed_                                                         |
```
Comment (matthewmayer) [inline docs/guide/upgrading.md]: i think you mean faker.image.lorempixel?

Maybe "Removed, as the LoremPixel service is no longer available"?

**Review by matthewmayer — CHANGES_REQUESTED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by matthewmayer** [comment]
The title of this PR is also incorrect

**Comment by Shinigami92** [comment]
> The title of this PR is also incorrect

Thank you @matthewmayer, your reviews are really welcome :heart:

**Review by xDivisionByZerox — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Pull request merged**